control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -2733,3 +2751,13 @@ func (r *reconciler) reconcileAzureCloudNodeManager(ctx context.Context, image s
 
 	return errs
 }
+
+// imageRegistryPlatformWithPVC returns true if the platform requires a PVC for the image registry.
+func imageRegistryPlatformWithPVC(platform hyperv1.PlatformType) bool {
```

Comment:
IBM Cloud may also use a PVC for the image registry.

Reply:
I'm not sure about that:

https://github.com/openshift/cluster-image-registry-operator/blob/f624d07b8ab8a01f0fa07880caafb65da9239656/pkg/storage/storage.go#L197-L198


And https://github.com/openshift/cluster-image-registry-operator/blob/master/pkg/storage/ibmcos/ibmcos.go#L61
I don't see the need of bootstrap here.

I'll defer to @csrwng who might know better on this.

Reply:
IBM Cloud has various deployment models that we need to handle.  We'll work with @csrwng.

Reply:
Good to know. Yeah talk to Cesar and feel free to add IBMCloud to the list of platform where CIRO bootstrap needs to happen first.

Reply:
@rtheis given that this already went in specifically for OpenStack on main, let's create a separate issue for IBM Cloud if you are running into issues as well.